src/components/learner-credit-management/BudgetDetailAssignments.jsx

```diff
@@ -28,7 +34,7 @@ const BudgetDetailAssignmentsHeader = ({
     }
   }, [navigate, location, locationState]);
 
-  if (isRetired) {
+  if (status === BUDGET_STATUSES.retired || status === BUDGET_STATUSES.expired) {
```

Comment:
[nit] I believe you can use utilize your newly minted utils function `isBudgetInactive`

``` js
export const isBudgetInactive = (status) => (
  status === BUDGET_STATUSES.retired || status === BUDGET_STATUSES.expired
);
```